### PR TITLE
Initial files added for deploy to heroku on sucessfule develop build

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: app/AppKernel.php [hunk 1]
```diff
@@ -30,6 +30,7 @@ public function registerBundles()
             $bundles[] = new Symfony\Bundle\WebProfilerBundle\WebProfilerBundle();
             $bundles[] = new Sensio\Bundle\DistributionBundle\SensioDistributionBundle();
             $bundles[] = new Sensio\Bundle\GeneratorBundle\SensioGeneratorBundle();
+            $bundles[] = new \Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle();
```
Comment (eddiejaoude) [inline app/AppKernel.php]: Do you need the prepended `\` ?

File: composer.json [hunk 2]
```diff
@@ -47,6 +47,11 @@
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile",
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::prepareDeploymentTarget"
+        ],
+        "compile" : [
+            "app/console cache:clear",
+            "app/console doctrine:schema:create",
+            "app/console doctrine:fixtures:load --no-interaction"
```
Comment (eddiejaoude) [inline composer.json]: :+1:

File: app/AppKernel.php (hunk 1 above)
Comment (brettminnie) [inline app/AppKernel.php]: Good spot

**Comment by eddiejaoude** [comment]
:+1:

**Pull request merged**